ftw/zipexport/browser/zipexportview.py

```diff
@@ -3,6 +3,7 @@
 from ftw.zipexport.generation import NotEnoughSpaceOnDiskException
 from ftw.zipexport.generation import ZipGenerator
 from ftw.zipexport.interfaces import IZipRepresentation
+from plone.protect.interfaces import IDisableCSRFProtection
```

Comment:
Also be aware that a lot of our deployments use `plone.protect` [2.0.2](https://matrix.4teamwork.ch/deployments?utf8=%E2%9C%93&deployment_filter%5Bname%5D=&deployment_filter%5Bmaintainer_id%5D=&deployment_filter%5Bkitt_id%5D=&deployment_filter%5Bdependencies_attributes%5D%5B0%5D%5Bname%5D=plone.protect&deployment_filter%5Bdependencies_attributes%5D%5B0%5D%5Bversion%5D=%3C3&deployment_filter%5Bdependencies_attributes%5D%5B0%5D%5Bnot%5D=0&deployment_filter%5Bdependencies_attributes%5D%5B1%5D%5Bname%5D=ftw.zipexport&deployment_filter%5Bdependencies_attributes%5D%5B1%5D%5Bversion%5D=&deployment_filter%5Bdependencies_attributes%5D%5B1%5D%5Bnot%5D=0&deployment_filter%5Brepository_url%5D=&deployment_filter%5Bstatus%5D=installed&deployment_filter%5Bdirty%5D=0&deployment_filter%5Bmissing_patches%5D=0&deployment_filter%5Bneeds_rotation%5D=0&deployment_filter%5Bunpacked_zodbs%5D=0&deployment_filter%5Bregex%5D=0&deployment_filter%5Brunning%5D=0&deployment_filter%5Bstopped%5D=0&deployment_filter%5Bstale%5D=0&deployment_filter%5Bfrom_src%5D=0), not [3.x](https://matrix.4teamwork.ch/deployments?utf8=%E2%9C%93&deployment_filter%5Bname%5D=&deployment_filter%5Bmaintainer_id%5D=&deployment_filter%5Bkitt_id%5D=&deployment_filter%5Bdependencies_attributes%5D%5B0%5D%5Bname%5D=plone.protect&deployment_filter%5Bdependencies_attributes%5D%5B0%5D%5Bversion%5D=%3E%3D3&deployment_filter%5Bdependencies_attributes%5D%5B0%5D%5Bnot%5D=0&deployment_filter%5Bdependencies_attributes%5D%5B1%5D%5Bname%5D=ftw.zipexport&deployment_filter%5Bdependencies_attributes%5D%5B1%5D%5Bversion%5D=&deployment_filter%5Bdependencies_attributes%5D%5B1%5D%5Bnot%5D=0&deployment_filter%5Brepository_url%5D=&deployment_filter%5Bstatus%5D=installed&deployment_filter%5Bdirty%5D=0&deployment_filter%5Bmissing_patches%5D=0&deployment_filter%5Bneeds_rotation%5D=0&deployment_filter%5Bunpacked_zodbs%5D=0&deployment_filter%5Bregex%5D=0&deployment_filter%5Brunning%5D=0&deployment_filter%5Bstopped%5D=0&deployment_filter%5Bstale%5D=0&deployment_filter%5Bfrom_src%5D=0). 

Although there is an `IDisableCSRFProtection` in the `2.x`-branch, there was none in the released version (`2.0.2`)[https://github.com/plone/plone.protect/blob/2.0.2/plone/protect/interfaces.py], which means that this change will break support for older Plone 4 versions (`<=4.3.7`).

Should we maybe make this optional?
Marking the request is only necessary when _automatic_ CSRF protection is enabled, which is not the case for those `plone.protect = 2.0.2` deployments. That means we could just not do anything if the interface cannot be imported. That's what we usually do.